tests/api2/test_twofactor_auth.py

```diff
@@ -30,19 +30,21 @@ def test_login_without_2fa():
         'username': TEST_USERNAME,
         'password': TEST_PASSWORD,
         'full_name': TEST_USERNAME,
-        'configure_twofactor_auth': True,
     }):
-        assert call('auth.login', TEST_USERNAME, TEST_PASSWORD) is True
+        assert call('auth.get_login_user', TEST_USERNAME, TEST_PASSWORD) is not None
```

Comment:
What difference does it make, how were the tests failing?

Reply:
The tests were failing because of unexpected user attribute which we had removed lately. For good measure, I just divided the logic to test the auth part only